test-script/gen_commands.py

```diff
@@ -0,0 +1,34 @@
+import glob
+import os
+import argparse
+
+modes = ['atomicity-violation', 'observability-gap', 'time-travel']
+mode_map = {
+    'atomicity-violation': 'atom-vio',
+    'observability-gap': 'obs-gap',
+    'time-travel': 'time-travel'
+}
```

Comment:
sieve accepts both `atom-vio` `obs-gap` and `atomicity-violation` `observability-gap`.